### PR TITLE
refactor(clint): whitespace/newline

### PR DESCRIPTION
**Pull request by lewis6991**

### CODE REVIEW

**Comment by zeertzjq** [comment]
What lint warnings does this fix?

File: src/nvim/api/vim.c [hunk 1]
```diff
@@ -252,17 +252,23 @@ void nvim_feedkeys(String keys, String mode, Boolean escape_ks)
...
+      insert = true;
+      break;
     case 'x':
-      execute = true; break;
+      execute = true;
+      break;
     case '!':
-      dangerous = true; break;
+      dangerous = true;
+      break;
```
Comment (zeertzjq) [inline src/nvim/api/vim.c:271]: What lint warnings does this fix?

Comment (lewis6991) [reply src/nvim/api/vim.c:271]: It fixes `whitespace/newline`. See: https://raw.githubusercontent.com/neovim/doc/gh-pages/reports/clint/errors.json 

They have `verbosity=0` which is why you might not have seen them.

We don't have to go with this change. I'm not sure if I'm a fan of it. I'm just trying to close out the remaining errors.

Comment (justinmk) [reply src/nvim/api/vim.c:271]: If switch-case formatting is covered by uncrustify, let's remove the clint rule. cc @dundargoc

Comment (lewis6991) [reply src/nvim/api/vim.c:271]: Uncrustify does cover this, it doesn't allow `case 'n': stmt; break;` as it requires a newline after `:`.

Comment (lewis6991) [reply src/nvim/api/vim.c:271]: Raised #20619

Comment (dundargoc) [reply src/nvim/api/vim.c:271]: There was a specific reason why I didn't add this fix in uncrustify, but I dont remember why. Can investigate.

**Pull request closed without merging**